corgidrp/photon_counting.py

```diff
@@ -150,7 +150,14 @@ def get_pc_mean(input_dataset, pc_master_dark=None, T_factor=None, pc_ecount_max
         raise ValueError('bin_size must be less than the number of frames in input_dataset.')
         
     num_bins = len(input_dataset)//bin_size 
-    
+
+    lines = []
+    for line in input_dataset[0].ext_hdr['HISTORY']:
+        lines += [line]
+    msg = 'Dark subtracted using dark'
+    if msg in lines:
+        pc_master_dark = None # dark subtraction was already done, so override any input pc_master_dark
```

Comment:
maybe also add a print statement to let the user know that we are skipping dark subtraction.